src/OPL3.cpp

```diff
@@ -139,17 +139,32 @@ byte OPL3::getChipRegisterOffset(short reg) {
 }
 
 
+/**
+ * Write a given value to a chip wide register.
+ *
+ * @param synthUnit - The chip to address [0, 1]
```

Comment:
This param should be removed since the base OPL3 class is not handling multiple chips.

Reply:
Oops, I was a bit too liberal with copy pasting from OPL3Duo